### ISSUE TITLE
Improve error logging

### ISSUE THREAD

**Issue by Jack-Dane** [Jack-Dane/odoo-wp-plugin#8]
**Is your feature request related to a problem? Please describe.**
When required fields are not entered into the form mappings there is no way to tell what went wrong. 

**Describe the solution you'd like**
An indication to the user that something went wrong when sending the request to Odoo.

**Comment by Jack-Dane** [comment]
Completed as part of #13

**Comment by Jack-Dane** [comment]
Hi @klacol, thanks for the kind words. 

In this story I added some logging which should log any errors on the errors page in the Odoo Forms sub-menu (they will be added to the table). However, when testing I did notice instances where the API returned OK but actually failed silently. So check the errors page to see if you have any error logs in the table. 

If there are no logs, check to make sure that all the required fields on the `crm.lead` model are included in the form submit. Also, it might be easier to debug the issue if you start with a few basic fields and then slowly add more until it fails. 

If you still can't get to the bottom of the issue, feel free to open a bug ticket and I can take a more in depth look into your specific use case.
